multi-agent/adapters/outbound/hitl/__init__.py

```
@@ -0,0 +1,19 @@
+"""
+from .channel_gate import ChannelApprovalGate
+from .auto_resolving_gate import AutoResolvingApprovalGate
+from .gate_factory import ChannelApprovalGateFactory
+from .redis_overrides_store import RedisOverridesStore
+
+__all__ = [
+    "ChannelApprovalGate",
+    "AutoResolvingApprovalGate",
+    "ChannelApprovalGateFactory",
+    "RedisOverridesStore",
+]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)